Editor/Module Management/ModuleInstaller.cs

```diff
@@ -69,34 +52,51 @@ private static void InstallModules()
 
         private static bool HasAnyMissingModule()
         {
-            var packages = GetPackageList();
-            var modules = ModuleList.Modules;
-            return modules.Select(module => packages.All(info => info.name != module.name)).FirstOrDefault();
+            var packageNames = GetPackageList().Select(p => p.name);
+            var moduleNames = ModuleList.Modules.Select(m => m.name);
+            
+            // returns true if any package name is missing in packages list
+            return moduleNames.Except(packageNames).Any();
+        }
+        
+        public static bool IsModuleInstalled(ModuleInfo module)
+        {
+            return GetPackageList().Any(info => info.name == module.name);
         }
 
         private static PackageCollection GetPackageList()
         {
             var listRequest = Client.List(true);
             while (!listRequest.IsCompleted)
                 Thread.Sleep(20);
-
+ 
             if (listRequest.Error != null)
             {
                 Debug.Log("Error: " + listRequest.Error.message);
                 return null;
             }
-
+            
             return listRequest.Result;
         }
 
-        private static void AddRequestPackage(string name)
+        public static void AddModule(string name)
```

Comment:
Personally, I prefer AddRequestPackage as it's more clear what it does. 
In general, I think it's kinda weird to have both an AddModule and InstallModule, As the name implies the same thing basically.